Source/player.cpp

```diff
@@ -2473,7 +2473,7 @@ BOOL PM_DoStand(int pnum)
 	return FALSE;
 }
 
-BOOL PM_DoWalk(int pnum)
+BOOL PM_DoWalk(int pnum) //Movement towards upleft, up, and upright

```

Comment:
Functions should be documented using comment blocks
```suggestion
/**
 * @brief Movement towards NW, N, and NE
 */
BOOL PM_DoWalk(int pnum)
```
Oh, and directions are given in cardinal and intercardinal directions.